includes/class-events.php

```diff
@@ -450,81 +360,30 @@ private function reset_event_lock( $event ) {
 	/**
 	 * Turn the event action into a string that can be used with a lock
 	 *
-	 * @param object $event Event data.
+	 * @param Event|stdClass $event
 	 * @return string
 	 */
-	public function get_lock_key_for_event_action( $event ) {
+	public function get_lock_key_for_event_action( $event ): string {
 		// Hashed solely to constrain overall length.
-		return md5( 'ev-' . $event->action );
-	}
-
-	/**
-	 * Mark an event completed, and reschedule when requested
-	 *
-	 * @param object $event Event data.
-	 */
-	private function update_event_record( $event ) {
```

Comment:
`$event->reschedule();` already encapsulates all this logic, so we can just drop it :)